src/main/include/subsystems/Climber.hpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2020 FRC Team 3512. All Rights Reserved.
+
+#pragma once
+
+#include <frc/DoubleSolenoid.h>
+#include <frc/Joystick.h>
+#include <frc/PowerDistributionPanel.h>
+#include <frc/Solenoid.h>
+#include <rev/CANSparkMax.h>
+
+#include "Constants.hpp"
+#include "communications/PublishNode.hpp"
+#include "subsystems/SubsystemBase.hpp"
+
+namespace frc3512 {
+
```

Comment:
Write a doxygen comment above the class saying what the climbing procedure is at a high level.